src/surge-xt/gui/SurgeGUIEditorValueCallbacks.cpp

```diff
@@ -2350,6 +2350,63 @@ int32_t SurgeGUIEditor::controlModifierClicked(Surge::GUI::IComponentTagValue *c
                                 });
                         }
                     }
+                    break;
+                    case ct_amplitude_ringmod:
+                    {
+                        contextMenu.addSeparator();
+                        auto dt = p->deform_type;
+
+                        Surge::Widgets::MenuCenteredBoldLabel::addToMenuAsSectionHeader(
+                            contextMenu, "COMBINATOR MODE");
```

Comment:
"RING MODDULATOR MODE"

Let's name it like this for now since the GUI refers to those sliders as RM and all the documentation etc.